src/olympia/api/paginator.py

```diff
@@ -21,6 +21,8 @@ def validate_number(self, number):
             raise PageNotAnInteger('That page number is not an integer')
         if number < 1:
             raise EmptyPage('That page number is less than 1')
+        elif number > 100000:
```

Comment:
Actually, reading https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html and search for "index.max_result_window" seems to be a more realistic limit that got implemented in ES 2.x as far as I can see? Maybe we should use that instead?